File: HRMS.Core/Models/Education.cs
namespace HRMS.Core.Models;

public class Education : Base
{
    public string ISCED { get; set; } = string.Empty;

    public int EQF { get; set; }

    public string Qualification { get; set; } = string.Empty;

    public string FinishedEducation { get; set; } = string.Empty;

    public string QualificationOld { get; set; } = string.Empty;

    public string FinishedEducationOld { get; set; } = string.Empty;
}

File: HRMS.Desktop/lib/models/education.dart
import 'package:json_annotation/json_annotation.dart';

part 'education.g.dart';

@JsonSerializable()
class Education {
  int id;
  String isced;
  int eqf;
  String qualification;
  String finishedEducation;
  String qualificationOld;
  String finishedEducationOld;

  Education(
    this.id,
    this.isced,
    this.eqf,
    this.qualification,
    this.finishedEducation,
    this.qualificationOld,
    this.finishedEducationOld,
  );

  factory Education.fromJson(Map<String, dynamic> json) =>
      _$EducationFromJson(json);

  Map<String, dynamic> toJson() => _$EducationToJson(this);
}
